packages/react-chat/src/components/Textarea/styled.ts

```diff
@@ -28,7 +28,7 @@ export const Container = styled(TextareaAutosize, {
   ...textareaStyles,
   ...textareaUniqueStyles,
   // TODO use tokens
-  padding: '9px $4 10px',
+  padding: '11px $4 11px',
```

Comment:
line height is 20px. To get 42px height and centered, it needs to be margin Y 11px @ninabondar

Reply:
let's review and merge the storybook build fix to avoid this in the future 😅  @DecathectZero 
https://github.com/voiceflow/react-chat/pull/100